content/en/docs/Writing policies/match-exclude.md

```diff
@@ -120,6 +120,14 @@ spec:
 ```
 
 As of Kyverno 1.5.0, wildcards are supported in the `kinds` field allowing you to match on every resource type in the cluster.
+Also, Wildcard supported at pol.spec.match.resources.selector.matchLabels in key as well as value of labels.
```

Comment:
What about `pol.spec.match.any(all).xxx.xxx` and `pol.spec.exclude.xxx.xxx`?

Can you also wrap paths with ``?

Reply:
got it. I miss any/all again. Will update.